UP

> There are CI internal errors (module updates):
> 
> ```
> === updating zcbor (modules/lib/zcbor):
> Cloning into '/__w/zephyr/modules/lib/zcbor'...
> --- zcbor: cloning from /repo-cache/zephyrproject/modules/lib/zcbor
> done.
> HEAD is now at 75d0880 tests: Update mps2 board name
> ERROR: update failed for projects: sof, tflite-micro, thrift
> Error: Error: failed to run script step: command terminated with non-zero exit code: Error executing in Docker Container: 1
> Error: Process completed with exit code 1.
> Error: Executing the custom container implementation failed. Please contact your self hosted runner administrator.
> ```

The CI issue was solved. 
Please continue the PR review.

--- a/doc/contribute/documentation/generation.rst
+++ b/doc/contribute/documentation/generation.rst
@@ -98,7 +98,7 @@ as described below:
 
       .. code-block:: console
 
-         pip install -r ~/zephyrproject/zephyr/doc/requirements.txt
+         pip install -U -r ~/zephyrproject/zephyr/doc/requirements.txt
 
       On Ubuntu Linux:
 
@@ -133,7 +133,7 @@ as described below:
 
       .. code-block:: console
 
-         pip install -r ~/zephyrproject/zephyr/doc/requirements.txt
+         pip install -U -r ~/zephyrproject/zephyr/doc/requirements.txt
 
       Use ``brew`` and ``tlmgr`` to install the tools:
 
@@ -149,7 +149,7 @@ as described below:
 
       .. code-block:: console
 
-         pip install -r %HOMEPATH$\zephyrproject\zephyr\doc\requirements.txt
+         pip install -U -r %HOMEPATH$\zephyrproject\zephyr\doc\requirements.txt
 
       Open a ``cmd.exe`` window as **Administrator** and run the following command:
 

